packages/sdk/src/utils/listener.ts

```diff
@@ -4,40 +4,42 @@ import {
   TransactionFulfilledEvent,
   TransactionPreparedEvent,
 } from "@connext/nxtp-utils";
-import { Contract, providers } from "ethers";
+import { TransactionManager } from "@connext/nxtp-contracts/typechain";
+import { providers } from "ethers";
 import { Evt } from "evt";
 
-import { NxtpSdkEvent, NxtpSdkEvents, TransactionCompletedEvent, NxtpSdkEventPayloads } from "../sdk";
-
 import { getTransactionManagerContract } from "./contract";
-// Define event types
+
+export const TransactionManagerEvents = {
```

Comment:
Do the sender/receiver ones need to be here?

Reply:
The way I'm thinking about this is the TransactionManagerListener class is directly tied to the contract so it mirrors the contract events exactly. At the higher SDK level, we consume these events and translate them into the SDK specific events which are sender/receiver.